### ISSUE TITLE
[feat] supports STOMP over TCP when used in NodeJS environment

### ISSUE THREAD

**Issue by davidkhala** [stomp-js/stompjs#500]
From the FAQ, we know that
> **Q: Are there any missing features from Jeff Mesnil's stompjs?**
> The Jeff Mesnil’s stompjs supports STOMP over TCP when used in NodeJS environment. Other than this, there are no known missing functionality. If you intend to use this feature, please raise an issue.

Now I encounter an scenario that need a support in raw TCP STOMP support in stomp.js

The OCI (Oracle Cloud Infra) release OCI Queue, a new cloud-native queue service. It provides built-in stomp interface exposed port 61613. And in the recent roadmap, no plan to support websocket.

So if we are not using OCI SDK to manipulate, raw stomp is the only choice.

Please evaluate and consider if it is possible.

**Comment by kum-deepak** [comment]
It should definitely be possible. I am currently busy with the v7 release of this library. I will take it up after that.

**Comment by ertl** [comment]
TCP support would also help us a lot.

**Comment by kum-deepak** [comment]
I just now released the beta3, and it has support for Stomp over TCP. 

```javascript
import { TCPWrapper } from '@stomp/stompjs';
// or
// The UMD does not contain the TCPWrapper, as browsers do not need it
const TCPWrapper = require('@stomp/stompjs/bundles/tcp-wrapper.cjs').TCPWrapper;
//...
client.webSocketFactory = () => new TCPWrapper('127.0.0.1', 61613); // host and port
```

I am happy to see that it passes the entire test suite. I have added this mode in regular GitHub CI tests as well.

Give it a spin and share your results.

**Comment by davidkhala** [comment]
@kum-deepak Hi Deepak, I find the error in 7.0-beta4

```
import {TCPWrapper} from '@stomp/stompjs';
        ^^^^^^^^^^
SyntaxError: The requested module '@stomp/stompjs' does not provide an export named 'TCPWrapper'
```
Any idea on why it is missing on my side?

**Comment by kum-deepak** [comment]
Moved it to an independent npm package. Please see https://github.com/stomp-js/tcp-wrapper

**Comment by kum-deepak** [comment]
Released 7.0,0
